### DISCUSSION TITLE
JSBSim simulation in the loop with PX4

### DISCUSSION THREAD

**Discussion by oricabel** [JSBSim-Team/jsbsim#639] [Q&A] [answered]
Hello all, i am doing a project as a part of my studies, and I wanted to know if it is possible to use the JSBSim integration with PX4 to simulate different aircrafts? I already managed to run the simulation with QGroundControl with the default aircraft (Rascal), and I would like to know if its possible for me to use my aircraft's configuration files (which I already have)to simulate my aircraft..

I would really appreciate if any one can tell me whether or not its possible.

Thanks

**Comment by Jaeyoung-Lim** [comment, marked as answer]
@oricabel The px4 jsbsim repository(https://github.com/Auterion/px4-jsbsim-bridge) already contains 4 models so of course it is possible to use/add different aircraft models 

You just need to map the actuator outputs from PX4 properly to your model

**Reply by oricabel** [reply]
Thank you so much for the reply! I am now trying to build the jsbsim_bridge, but having some trouble... I'll ask about it.
